blaze/expr/broadcast.py

```diff
@@ -99,8 +100,8 @@ def scalar_symbols(exprs):
     return scalars
 
 
-Broadcastable = (Arithmetic, Math, Map, Field, DateTime)
-WantToBroadcast = (Arithmetic, Math, Map, DateTime)
+Broadcastable = (Arithmetic, Math, Map, Field, DateTime, USub)
+WantToBroadcast = (Arithmetic, Math, Map, DateTime, USub)
```

Comment:
That bit of the class hierarchy could definitely be improved.  Not sure what the right way is.